app/routers/admin.py

```
@@ -58,6 +65,29 @@ def verify_admin(credentials: HTTPBasicCredentials = Depends(security)):
         )
     return credentials.username
 
+def verify_admin_action(
+    request: Request,
+    credentials: HTTPBasicCredentials | None = Depends(security),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)